<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 1 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_number1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_number2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Sumup">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sum up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_substraction">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Substraction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_division">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Division</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multiplication">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiplication</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_result">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
